robot/basestation/static/js/roslibjs/ros-helpers.js

```diff
@@ -188,6 +204,76 @@ function initRosWeb () {
 }
 
 /* functions used in main code */
+
+function rosTimestamp () {
```

Comment:
When adding any new function good practice is to add a docstring for it.
Please add a high level overview of what each function does, in a sentence or two.

Reply:
this is resolve

Reply:
@PeterGhimself It looks like I did add a docstring. Did I do it right?